mlflow/entities/trace_status.py

```diff
@@ -17,3 +19,19 @@ def to_proto(self):
     @staticmethod
     def from_proto(proto_status):
         return ProtoTraceStatus.Name(proto_status)
+
+    def to_otel_status(self) -> trace_api.Status:
+        return trace_api.Status(_MLFLOW_STATUS_CODE_TO_OTEL[self])
```

Comment:
nit/question: should we map `IN_PROGRESS` to `UNSPECIFIED`? what happens at the moment if we try to call this function when the value is `IN_PROGRESS`?

Reply:
Oh great catch! Actually we don't need `to_otel_status()`, we only do the opposite mapping. Will remove this function.